Enhancement for the retrieve file generic (#32035)
demisto/content: Packs/MicrosoftDefenderAdvancedThreatProtection/ReleaseNotes/1_16_26.md

#### Playbooks

##### MDE - Retrieve File

> - Added transformer that allows to use this playbook as part of malware investigation or as standalone playbook.

Reviewer: ```suggestion
Added a transformer that allows using this playbook as part of a malware investigation or as a standalone playbook.
```
